projects/novo-elements/src/elements/form/FieldInteractionApi.ts

```diff
@@ -561,6 +562,21 @@ export class FieldInteractionApi {
       this.triggerEvent({ controlKey: key, prop: 'pickerConfig', value: args });
     }
   }
+
+  addPropertiesToPickerConfig(key: string, properties: { [key: string]: unknown }) {
+    let control = this.getControl(key);
+    if (!control || control.restrictFieldInteractions) {
+      return;
+    }
+
+    const config = {
+      ...(control && control.config),
```

Comment:
The use of `control &&` seems unnecessary, since you have the `if (!control ||` above.

Reply:
good catch!